Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue5046.xaml.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+using Xamarin.Forms.Xaml;
+
+namespace Xamarin.Forms.Controls.Issues
+{
+#if APP
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 5046, "CornerRadius doesn't work in explicit style when implicit style exists", 
+	       PlatformAffected.Android | PlatformAffected.iOS)]
+	[XamlCompilation(XamlCompilationOptions.Compile)]
+	public partial class Issue5046 : ContentPage
```

Comment:
If you really want to go all out you could add a UI test for this. Change the base page to a `TestContentPage` and add a test that verifies this. There are a lot of examples to be found in the project. Then again, you already have a unit test so not absolutely necessary.

Reply:
I tried, but I didn't manage to assert the CornerRadius in these UI test cases. With RunningApp.Query I can only get an AppResult object with a very limited set of properties.  
All the other UITests I looked at also only verify that a component exists, or that the app doesn't crash and not a specific property of one component.
But I think the unit test should be enough (?) otherwise it would be testing the same thing twice anyway

Reply:
It's fine for now with the unit tests. Did you see the .Property (from the top of my head) method? I think with that you should be able to query all properties on an object :)